Sources/XMLCoder/Auxiliaries/XMLCoderElement.swift

```diff
@@ -176,8 +180,9 @@ struct XMLCoderElement: Equatable {
         }
 
         var string = ""
-        string += element._toXMLString(indented: level + 1, escapedCharacters, formatting, indentation)
-        string += prettyPrinted ? "\n" : ""
+        let indentLevel = !isInlided ? level + 1 : level
```

Comment:
Reverse the ternary, making it easier to parse

```suggestion
        let indentLevel = isInlined ? level : level + 1
```

Reply:
done